packages/deployer/src/applyConfig.ts

```diff
@@ -0,0 +1,8 @@
+import { applyDeploymentConfig } from './deployUtils'
```

Comment:
Don't you want to export these scripts as CLI commands? Why nort?

Reply:
this is an ethers/hardhat script, like deploy. 
it also depends on the "deployments" created by deploy under this folder.